glossary.tex

```diff
@@ -46,7 +46,7 @@
 \newglossaryentry{L1d}
 {
   name={L1d},
-  description={Level 1 data cache, often called \textbf{primary cache}, is a static memory integrated with processor core that is used to store data recently accessed by a processor and also data which may be required by the next operations.},
+  description={Level 1 data cache, of$ten called \textbf{primary cache}, is a static memory integrated with the CPU core that is used to store data recently accessed by a processor and also data which may be required by the next operations},
```

Comment:
What's with the `$` in `of$ten`?

`Processor` should also be mentioned in the glossary, as a synonym for 'CPU'?

We usually talk about a "multi-core processor" not a "multi-core CPU" (since the 'unit' part of CPU is confusing w.r.t. a core which is a single compute 'unit').

Reply:
Thanks for the remarks.
I prefer CPU, because processor is more general, it can mean GPU as well as CPU. It's clear that we mean CPU here.
I ack the possible confusion, that is why I explained in the glossary that a CPU can contain more than one core.
If you agree, I will make the distinction between processor and CPU in the glossary.

Reply:
I never looked at it that way, but I guess you're right, strictly speaking. When we say 'processor' it's usually a synonym for CPU.

I can see why you would like to make the distinction and would use 'processor' as an umbrella  term for CPU/GPU though, so I'm fine with that. As long as we use it consistently.

Reply:
OK, I will make sure it is consistent throughout the tutorial.